example_app/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: ..
   specs:
-    packer (0.4.2)
+    packer (0.5.0)
```

Comment:
i would be cautious about doing dependency updates at the same time as a functionality change. Unless, of course, the update is necessary for a particular feature change.

Reply:
@tgallacher I updated the example app at the same time (replaced the npm lockfile with the yarn lockfile) so I think it makes sense for it to use the updated version. It's a path based gem dependency too, pointing to the parent directory, it doesn't install it from the gem repository.